src/App.tsx

```diff
@@ -14,11 +15,136 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+const numArr: number[] = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+
+type State = {
+  visibleGoods: boolean,
+  copyArrGoods: string[],
+  secondCopyArr: string[],
```

Comment:
do you need copyArrGoods and secondCopyArr?
I believe you can use only one array for goods in state